# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/shaderc/hotfix-version.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- shaderc-2024.0/glslc/test/CMakeLists.txt.vanilla	2024-04-22 19:22:58.358000000 +0200
+++ shaderc-2024.0/glslc/test/CMakeLists.txt	2024-04-22 19:23:08.086000000 +0200
@@ -23,6 +23,6 @@
   add_test(NAME glslc_tests
     COMMAND ${Python_EXECUTABLE}
     ${CMAKE_CURRENT_SOURCE_DIR}/glslc_test_framework.py
-    $<TARGET_FILE:glslc_exe> $<TARGET_FILE:spirv-dis>
+    $<TARGET_FILE:glslc_exe> spirv-dis
     --test-dir ${CMAKE_CURRENT_SOURCE_DIR})
 endif()
--- shaderc-2024.0/glslc/CMakeLists.txt.vanilla	2024-04-22 19:23:46.469000000 +0200
+++ shaderc-2024.0/glslc/CMakeLists.txt	2024-04-22 19:23:54.041000000 +0200
@@ -53,7 +53,6 @@
 target_include_directories(glslc_exe PRIVATE ${CMAKE_CURRENT_BINARY_DIR}/.. ${spirv-tools_SOURCE_DIR}/include)
 set_target_properties(glslc_exe PROPERTIES OUTPUT_NAME glslc)
 target_link_libraries(glslc_exe PRIVATE glslc shaderc_util shaderc)
-add_dependencies(glslc_exe build-version)
 
 shaderc_add_tests(
   TEST_PREFIX glslc
--- shaderc-2024.0/glslc/src/build-version.inc.vanilla	2024-04-22 19:27:11.533000000 +0200
+++ shaderc-2024.0/glslc/src/build-version.inc	2024-04-22 19:28:42.874000000 +0200
@@ -0,0 +1,3 @@
+"shaderc 2024\n"
+"SPIRV-Tools 2024\n"
+"glslang 2024\n"
